sdk/textanalytics/perf-tests/text-analytics/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "../../../../tsconfig.package",
+  "compilerOptions": {
+    "target": "es5",
```

Comment:
I don't really have any objection to anything in here, but are we actually using the compiled JS for anything? And why es5? Storage targets ES5 for reasons, but we should be able to safely target ES6 in cognitive. The NPM script uses ts-node, anyway, though. Maybe a `noEmit` configuration that just does typechecking would be the most appropriate thing.

Reply:
Done. I am also targeting `DOM` for now because of https://github.com/Azure/azure-sdk-for-js/issues/14069.